ci-operator/jobs/sclorg/ansible-tests/sclorg-ansible-tests-master-periodics.yaml

```diff
@@ -0,0 +1,87 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 6 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: sclorg
+    repo: ansible-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: dotnet-ocp4.13-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.13-lp-interop-dotnet-interop-aws
```

Comment:
If you add the line @ascerra [mentioned earlier](https://github.com/openshift/release/pull/37713#discussion_r1196992211) and run `make update`, this will be done for you.